src/dcap_provider.cpp

```diff
@@ -54,12 +54,15 @@ static const std::map<std::string, std::string> default_values = {
 // New API version used to request PEM encoded CRLs
 constexpr char API_VERSION_LEGACY[] = "api-version=2018-10-01-preview";
 constexpr char API_VERSION[] = "api-version=2020-02-12-preview";
-
 static char DEFAULT_CERT_URL[] =
-    "https://global.acccache.azure.net/sgx/certificates";
+    "https://global.acccache.azure.net/sgx/certificates/";

```

Comment:
Once THIM Service deploys new controller to align with Intel, this should be updated. The final URL will be like: 
`https://global.acccache.azure.net/sgx/certification/v3/pckcert?qeid={}&cpusvn={}&pcesvn={}&pceid={}&encrypted_ppid={}`

Reply:
Agreed! I coded this based on the current prod code. Will update this